ParcialProgramacion4/src/Ejecutora.java

```diff
@@ -0,0 +1,51 @@
+import java.util.Scanner;
+
+public class Ejecutora {
+    public static void main(String [] args){
+        Scanner s = new Scanner(System.in);
```

Comment:
usemos nombres completos
```suggestion
        Scanner scanner = new Scanner(System.in);
```